Multisig/UI/App/HeaderViewController/HeaderViewController.swift

```diff
@@ -103,7 +103,9 @@ final class HeaderViewController: ContainerViewController {
             if let safe = selectedSafe {
                 safeBarView.setAddress(safe.addressValue)
                 safeBarView.setName(safe.displayName)
-                safeBarView.setReadOnly(safe.isReadOnly)
+                if updateReadOnlyLabel {
+                    safeBarView.setReadOnly(safe.isReadOnly)
```

Comment:
what if instead of this argument we would just change the default `isReadOnly` value to `false`?

In the implementation of that isReadOnly we need to check that if `ownersInfo` is nil, then return false; otherwise decide based on the implemented logic.

Reply:
ok